.github/workflows/ci-test.yml

```diff
@@ -40,6 +41,12 @@ jobs:
         uses: ./.github/actions/changes
         with:
           workflow-file: ci-test.yml
+      - id: need-build-images
+        name: find changes need image build.
+        uses: ./.github/actions/matches
+        with:
+          pattern: '^documentation\|^.circleci'
```

Comment:
We can skip for developer site changes in `developers.diem.com`

Reply:
good call, lets put this everywhere in prepare.

Reply:
I will do this update

Reply:
Also, `.circleci` can be deprecated after today's cleanup.